Tests/CryptoSwiftTests/XCTestManifests.swift

```diff
@@ -13,7 +13,7 @@ extension AESCCMTests {
         ("testAESCCMTestCase3", testAESCCMTestCase3),
         ("testAESCCMTestCase3Decrypt", testAESCCMTestCase3Decrypt),
         ("testAESCCMTestCase3DecryptPartial", testAESCCMTestCase3DecryptPartial),
-        ("testAESCCMTestDVPT256", testAESCCMTestDVPT256),
```

Comment:
It will back as this is how it's autogenerated by `swift test --generate-linuxmain`